sweep/sweeper.go

```diff
@@ -157,6 +158,19 @@ func (s SweepState) String() string {
 	}
 }
 
+// RBFInfo stores the information required to perform a RBF bump on a pending
+// sweeping tx.
+type RBFInfo struct {
```

Comment:
To be _fully_ aware of all the RBF rules, don't we also need to track information related to the set of inputs in the publishing transaction? As an example: if the inputs are unconfirmed, if a new unconfirmed input was added in a recent iteration, etc. 

Though part of me wonders if we can just get away more with the existing spray-and-pray approach...

Reply:
yeah so the idea is, since we have txid, we can fetch the tx and get its inputs. meanwhile we have a set of inputs tracked in the sweeper, so can then use `tx.TxIn` to decide the starting state of the inputs in the sweeper.

> Though part of me wonders if we can just get away more with the existing spray-and-pray approach...

Def, we just relentlessly call `testmempoolaccept`, not feasible for neutrino tho.